src/scripts/helpers/handlers/analytics.coffee

```diff
@@ -14,18 +14,25 @@ define (require) ->
         l: Date.now()
 
     # Send the current page to every analytics service
-    send: (account, fragment = Backbone.history.fragment) ->
-      if not /^\//.test(fragment) then fragment = '/' + fragment
+    sendAnalytics: (accounts, fragment = Backbone.history.fragment) ->
+
+      # temporarily support a single field for analytics (sometimes it is `null`)
+      unless Array.isArray(accounts)
+        console.log 'Consider calling analytics.sendAnalytics with an Array'
+        @sendAnalytics([accounts], fragment)
 
       require ['analytics'], (ga) =>
-        # Use the default analytics ID in settings if no account is specified
-        account ?= settings.analyticsID
 
-        # TODO investigate if we need specific names for our tracker name
-        trackerName = @getTrackerName(account)
-        ga('create', account, 'auto', trackerName)
+        accounts.forEach (account) =>
+          if not /^\//.test(fragment) then fragment = '/' + fragment
+          # Use the default analytics ID in settings if no account is specified
+          account ?= settings.analyticsID
```

Comment:
This is never reached if accounts above is empty, is it? I think it's not possible. So the site-wide code will never be sent. Tracing through currently-released production, I see this send method being called twice on each page navigation - once w/ "undefined" account, and once w/ the book-defined account code.